### DISCUSSION TITLE
how to enhance trained module

### DISCUSSION THREAD

**Discussion by franklydog** [JohnSnowLabs/spark-nlp#2746] [General]
Hi,
   I have trained a module based on USE and ClassifierDL.  It work well on most of time.But some times it can't work as I think.so I want to enhance it's accurate with new data based on trained module and don't train it from scratch. Would you give me some advice? Thanks.

**Comment by maziyarpanahi** [comment]
I converted this into a discussion since it's not really a feature request. I think the best is you prepare a Google Colab from top to bottom including the dataset (if it's public) and share it here. Improving accuracy in DL-based models is always a try-and-try more situation. So there is no one recommendation rather than just keep trying.

But sharing that Colab notebook can help others to start tweaking until it is improved more than the baseline.

**Comment by franklydog** [comment]
Can I load the trained model first, and then continue to train it with another data? Instead of starting from scratch every time.

**Reply by maziyarpanahi** [reply]
No, every training starts from scratch

**Reply by franklydog** [reply]
Thanks
